components/rmrk/Create/CreateToken.vue

```diff
@@ -106,6 +106,7 @@ import { unwrapSafe } from '@/utils/uniquery'
 import { toNFTId } from '../service/scheme'
 import { usePreferencesStore } from '@/stores/preferences'
 import { Ref as RefType } from 'vue'
+import { Royalty } from '@/utils/royalty'
```

Comment:
build failed after sync with `main` branch. remove this to fix that

Reply:
yy. just removed